ocs_ci/ocs/hsbench.py

```diff
@@ -63,7 +62,6 @@ def create_resource_hsbench(self):
             namespace=self.namespace,
             pod_name=self.pod_name,
             pvc_name=self.pvc_obj.name,
-            node_name=node_name,
```

Comment:
if https://github.com/red-hat-storage/ocs-ci/pull/4240/files is merged earlier, this PR will need a rebase.

Reply:
Yes, a rebase needed here.